cmake/graph-compiler.cmake

```
@@ -4,6 +4,8 @@ if (NOT DEFINED GRAPH_COMPILER_LIBS)
     # versions, using find_package() first. If the package is not found, then using FetchContent.
     if (CMAKE_VERSION VERSION_LESS "3.24")
         find_package(GraphCompiler QUIET)
+    else ()
+        set(GC_FETCH_CONTENT_ARGS FIND_PACKAGE_ARGS NAMES GraphCompiler)
```

```suggestion
        set(GC_FETCH_CONTENT_ARGS "FIND_PACKAGE_ARGS NAMES GraphCompiler")
```
I think it's required here

---

no, it doesn't work if it's wrapped into a string